app/templates/partials/summary/summary.html

```diff
@@ -7,7 +7,9 @@
                 <h2 class="summary__title neptune" id="{{ group.id }}">{{ group.title }}</h2>
             {%- endif -%}
 
-            <div class="summary__block u-mb-l">
+            {%- if group.open_summary_block -%}
```

Comment:
I think this should be changed (sorry for not including this comment in the first pass!).
It looked odd to me when I first saw it but didn't look into it too deeply.  But after looking, there isn't a single instance where one is true and the other is false.
Also, theres a chance of an open without a close which would produce invalid HTML.

Is there a reason for this that I'm missing?

Reply:
There is one instance where they will be different.

https://github.com/ONSdigital/eq-survey-runner/pull/1789/files/8e821331540515439c1bcdf7e5d87f4c058450cb#diff-41b3143304060f6f5310f1cdb4d18153L7

If we use a single flag, then we get a double line:
![summary - test repeat until 1](https://user-images.githubusercontent.com/205209/45947358-d36a9f00-bfeb-11e8-88fb-9b5e6992b57a.png)
Versus the current functionality:
![summary - test repeat until 2](https://user-images.githubusercontent.com/205209/45947364-dd8c9d80-bfeb-11e8-8441-92dfc6406471.png)

I do agree that it's not great, will investigate if there's a more robust method.